physics/satmedmfvdifq.F

```diff
@@ -97,6 +99,10 @@ subroutine satmedmfvdifq_run(ix,im,km,ntrac,ntcw,ntiw,ntke,       &
      &                     prsi(ix,km+1), del(ix,km),                   &
      &                     prsl(ix,km),   prslk(ix,km),                 &
      &                     phii(ix,km+1), phil(ix,km)
+      real(kind=kind_phys), intent(inout) ::                            &
```

Comment:
are these always allocated? If not, use (:,:) instead of (ix,km)

Reply:
Thanks very much for fixing this, Sam! Will look at the code again on Monday and run the usual regression tests before merging.